projects/kit/components/tiles/tile-handle.directive.ts

```diff
@@ -47,9 +31,21 @@ export class TuiTileHandle {
         this.tile.onDrag([NaN, NaN]);
     }
 
-    @shouldCall(isDragging)
-    @HostListener('document:pointermove.silent', ['$event.x', '$event.y'])
+    @shouldCall(function (this: TuiTileHandle): boolean {
+        return !Number.isNaN(this['x']);
+    })
```

Comment:
I am not sure that we actually need this change

Reply:
fixed